src/main/java/teammates/common/datatransfer/questions/FeedbackRankRecipientsQuestionDetails.java

```diff
@@ -238,6 +239,79 @@ public String getQuestionAdditionalInfoHtml(int questionNumber,
                         Slots.QUESTION_ADDITIONAL_INFO, additionalInfo);
     }
 
+    private List<FeedbackResponseAttributes> getActualResponses(
```

Comment:
`response` param in `getQuestionResultStatisticsHtml` already contains the response of the `question`?

Reply:
`response` param contains only limited number of responses sufficient to show responses received by student but not sufficient to build statistics.

Reply:
Then please give a better name. I would make more sense to add this method in `FeedbackSessionResultsBundle.java`. In that case, please also add some unit tests.
`getAllResponsesForQuestion(question)`